src/Components/ListenOn/ListenOn.js

```diff
@@ -20,27 +21,82 @@ export default function ListenOn() {
                 </h3>
 
                 <span className="listenOnChannel">
-                    <img className="logo" src={appleLogo} alt="apple logo" />
-                    APPLE PODCASTS
+                    <a
+                        href="https://podcasts.apple.com/us/podcast/hopped-up-gaming-east/id797058638"
```

Comment:
I would probably make an array of objects in this file and have the objects contain the name, link and logo. that way you can fill these in programmatically. Plus it will be easier to change in the future

You could then make each `<span>` into it's own component called like `<ListenLink>` or something and pass the details from the objects in as props